x-pack/legacy/plugins/siem/public/pages/case/components/user_list/index.tsx

```diff
@@ -50,7 +57,8 @@ const renderUsers = (users: ElasticUser[]) => {
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
         <EuiButtonIcon
-          onClick={() => {}} // TO DO
+          data-test-subj="user-list-email-button"
+          onClick={handleSendEmail.bind(null, email)} // TO DO
```

Comment:
To do what? :)

Reply:
oh no longer a todo! im a green build tho so ill remove it in a future PR